### DISCUSSION TITLE
Creating a light version of the website for mobile responsive

### DISCUSSION THREAD

**Discussion by FarahFI** [TDRA-ae/aegov-dls#16] [General]
Will creating a light version of the website for mobile responsive be against the DLS 2.0?
The light version will only have the website's most important pages such as services, healthcare facilities and Contact Us page.
A pop-up will appear for the user once browsing the website from a mobile that says "Would you like to switch to the mobile light version of the website? However, the user will still be able to go back to the full version.
Please share your feedback on this

**Comment by moiz-conceptlz** [comment]
Hello @FarahFI 

To understand this better, let us first consider the assumption.

1. A perfectly responsive website already works accurately on mobile. However, you are planning to create a version of the website that is limited in functionality and only provides necessary functions.

2. You want to show potential users who visit the website from a mobile device a popup message that directs them to this light version as a choice.

The purpose and reason to create a limited (or a light version) of the website is a business decision. The design system does restrict such a feature. The purpose of the design system is to create consistency in terms of UI, accessibility, experience and content. What this means is that if it is absolutely necessary for a business to follow such an approach of creating a light version, the design system would not have an objection as long as the UI elements of the light version are consistent with the design system.
